src/task/serverTaskTerminal.ts

```diff
@@ -139,11 +169,23 @@ export class ServerTaskTerminal
 	 */
 	public serverWillBeStopped(): void {
 		this.writeEmitter.fire(
-			`This task will finish now, but the server will stay on since you've used the embedded preview recently.\r\n`
+			localize(
+				'taskFinished',
+				`This task will finish now, but the server will stay on since you've used the embedded preview recently.\r\n`
+			)

```

Comment:
```suggestion
			localize(
				'taskFinished',
				'This task will finish now, but the server will stay on since you've used the embedded preview recently.'
			) + '\r\n'

```